src/Assignment-2/assignment2.html

```diff
@@ -14,6 +14,139 @@
 	<!-- Start your code from here -->
 	
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+	<div class="main-body">
+        
+        <!-- START Header -->
+        <div>
+            <header>
+                <div class="first-part">
+                    <div class="item">
+                        <img src="https://www.the10xacademy.com/assets/images/logo-color.png" alt="">
+                    </div>
+                    <div class="item">  
+                        <form action="">
+                            <input type="text" placeholder="Search" name="Search">
+                        </form>
+                    </div>
+                </div>
+                <div class="second-part">
+                    <nav class="nav">
+                        <div class="nav-link"><a href="https://github.com/mrunabhavsar28">My Github Link</a></div>
+                        <div class="nav-link"><a href="https://www.the10xacademy.com/">The 10xacademy</a></div>
+                        <div class="nav-link"><a href="https://www.hackthebox.com/">Hackthebox</a></div>
+                        <div class="nav-link"><a href="https://www.pexels.com/">Pexels</a></div>
+​
+                    </nav>
+                </div>
+            </header>
+        </div>
+        <!-- END Header -->
+    <hr>
+​
+        <!-- START BODY -->
+        <section class="main-section">
+            <div class="left-section">
+                    <aside>
+                        <ul>Section 1</ul>
+                        <ul>Section 2</ul>
```

Comment:
the use of ul tag is 

> 				<ul>
					<li>section 1</li>
					<li>section 2</li>
					<li>section 3</li>
					<li>section 4</li>
					<li>section 5</li>
					<li>section 6</li>
					<li>section 7</li>
					<li>section 8</li>
					<li>section 9</li>
					<li>section 10</li>
				</ul>

Reply:
fix this in aside tag